setuptools_rust/build.py

```diff
@@ -138,6 +138,11 @@ def build_extension(self, ext: RustExtension, target_triple=None):
 
         features = set(ext.features)
         features.update(rust_features(binding=ext.binding))
+        if ext.py_limited_api and ext.binding == Binding.PyO3 and platform.python_implementation() != 'PyPy':
```

Comment:
I don't think we need to check about PyPy here. PyO3 will warn if using PyPy is inappropriate, and PyPy will support in a future version.

Reply:
(fixed in my PR)